packages/applications-service-api/src/middleware/validator/validateGetDocumentWithTypeRequestOpenAPI.js

```diff
@@ -0,0 +1,29 @@
+const { createRequestValidator } = require('../../utils/openapi');
+const ApiError = require('../../error/apiError');
+
+const validateGetDocumentWithTypeRequestOpenAPI = (req, res, next) => {
```

Comment:
This was intentionally done so the raw query would be used in the controller. This should avoid any potential mistakes / bugs of thinking why is it all caps etc. 

happy to change it but we thought this could lead to confusion / potential bugs in the future.